scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
lib/ and bin/ or scripts/ indicate a type of usage: generators/… is unknown, generators/lib/… are obviously `require()`d and generators/bin/… are obviously used directly from the command-line and not some other way around (better also add shebangs to generators which are missing that).

(Tests may also follow something like this, I remember myself accidentally running helpers.lua by busted as a test a few times before I wrote run_…_test helper shell functions.)